### PR TITLE
Upon clicking on the logo the user is redirected to the coding progress table

### PR DESCRIPTION
**Pull request by elvinmworia**

### CODE REVIEW

**Review by IsaackMwenda — APPROVED** [review]
Thanks @elvinmworia

File: public/index.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <!-- NAVBAR WITH DROPDOWN & RESPONSIVE TOGGLE -->
     <nav class="navbar navbar-expand-sm navbar-brown bg-white mb-1 py-0">
         <div class="container">
-            <a class="navbar-brand" href="#">
+            <a class="navbar-brand" id="homepage" href="#">
```
Comment (DanielMwendwa) [inline public/index.html]:
```suggestion
            <a class="navbar-brand" id="homepage-link" href="#">
```

File: public/js/controllers/ui_controller.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ export class UIController {
             mainContainer: "#main-container",
             activeLinkClassName: "active-link",
             activeLinks: "a.active-link",
+            homeLink: "#homepage",
```
Comment (DanielMwendwa) [inline public/js/controllers/ui_controller.js]:
```suggestion
            homePageLink: "#homepage-link",
```

File: public/js/controllers/app_controller.js [hunk 3]
```diff
@@ -19,6 +19,9 @@ class Controller {
         document
             .querySelector(Controller.DOMstrings.systemsMenu)
             .addEventListener("click", Controller.navigateToSelectedSystem);
+        document
+            .querySelector(Controller.DOMstrings.homeLink)
```
Comment (DanielMwendwa) [inline public/js/controllers/app_controller.js]:
```suggestion
            .querySelector(Controller.DOMstrings.homePageLink)
```

**Review by DanielMwendwa — APPROVED** [review]

**Pull request merged**